packages/app-store/cal-ai/api/add.ts

```diff
@@ -1,16 +1,56 @@
-import { createDefaultInstallation } from "@calcom/app-store/_utils/installation";
-import type { AppDeclarativeHandler } from "@calcom/types/AppHandler";
-
-import appConfig from "../config.json";
-
-const handler: AppDeclarativeHandler = {
-  appType: appConfig.type,
-  variant: appConfig.variant,
-  slug: appConfig.slug,
-  supportsMultipleInstalls: false,
-  handlerType: "add",
-  createCredential: ({ appType, user, slug, teamId }) =>
-    createDefaultInstallation({ appType, userId: user.id, slug, key: {}, teamId }),
-};
-
-export default handler;
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
this works to create the api key and put it in creds - but app doesn't show installed on FE - even though it is.